get_started.md

```diff
@@ -16,17 +16,18 @@ packages available for the platform to use:
 
 |Platform|Gazebo Versions|
 |---|---|
-| Ubuntu 24.04 Noble | [Gazebo Harmonic](/docs/harmonic/install_ubuntu){.external} (recommended), (recommended if using ROS 2 Jazzy) and [Gazebo Ionic](/docs/ionic/install_ubuntu){.external}
+| Ubuntu 24.04 Noble | [Gazebo Jetty](/docs/jetty/install_ubuntu){.external} (recommended), [Gazebo Harmonic](/docs/harmonic/install_ubuntu){.external} (default in ROS 2 Jazzy) and [Gazebo Ionic](/docs/ionic/install_ubuntu){.external} (default in ROS 2 Jetty)
 | Ubuntu 22.04 Jammy | [Gazebo Harmonic](/docs/harmonic/install_ubuntu){.external} (recommended) and [Gazebo Fortress](/docs/fortress/install_ubuntu){.external} (recommended if using ROS 2 Humble)
-| Mac Ventura | [Gazebo Harmonic](/docs/harmonic/install_osx){.external} (recommended) and [Gazebo Fortress](/docs/fortress/install_osx){.external}
-| Mac Monterey | [Gazebo Harmonic](/docs/harmonic/install_osx){.external} (recommended) and [Gazebo Fortress](/docs/fortress/install_osx){.external}
+| Mac Sequoia arm64 | [Gazebo Jetty](/docs/jetty/install_osx){.external} (recommended), [Gazebo Ionic](/docs/ionic/install_osx){.external}, [Gazebo Harmonic](/docs/harmonic/install_osx){.external} and [Gazebo Fortress](/docs/fortress/install_osx){.external}
+| Mac Sonoma arm64/amd64 | [Gazebo Jetty](/docs/jetty/install_osx){.external} (recommended), [Gazebo Ionic](/docs/ionic/install_osx){.external}, [Gazebo Harmonic](/docs/harmonic/install_osx){.external} and [Gazebo Fortress](/docs/fortress/install_osx){.external}
```

Comment:
based on https://github.com/gazebosim/docs/issues/602#issuecomment-3445053712, here's how we could document which macOS versions we support

```suggestion
| Mac Sequoia arm64 (until August 2027) | [Gazebo Jetty](/docs/jetty/install_osx){.external} (recommended), [Gazebo Ionic](/docs/ionic/install_osx){.external}, [Gazebo Harmonic](/docs/harmonic/install_osx){.external} and [Gazebo Fortress](/docs/fortress/install_osx){.external}
| Mac Sonoma arm64/amd64 (until August 2026) | [Gazebo Jetty](/docs/jetty/install_osx){.external} (recommended), [Gazebo Ionic](/docs/ionic/install_osx){.external}, [Gazebo Harmonic](/docs/harmonic/install_osx){.external} and [Gazebo Fortress](/docs/fortress/install_osx){.external}
```